cpp/oneapi/dal/algo/decision_forest/common.cpp

```diff
@@ -40,7 +42,7 @@ class descriptor_impl : public base {
             min_observations_in_leaf_node = 5;
         }
         else {
-            static_assert("Unknown task");
+            static_assert(detail::is_valid_task_v<Task>);
```

Comment:
Can be moved to the beginning of the body

Reply:
fixed